userspace/libsinsp/filter.cpp

```diff
@@ -1209,6 +1240,50 @@ bool sinsp_filter_check::flt_compare(cmpop op, ppm_param_type type, void* operan
 				}
 			}
 			return false;
+		case PT_CHARBUFARRAY:
```

Comment:
Can you also update the comment block on lines ~1223-~1226, now that filterchecks *can* extract multiple values?